release/smoke-tests/data-prepper/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright OpenSearch Contributors
```

Comment:
It might help to put this in the `data-prepper-tar` subdirectory, or some other subdirectory to clarify that it is for the tar.gz smoke tests.